src/app/components/Buttons/BlueBtn.tsx

```diff
@@ -0,0 +1,9 @@
+function BlueBtn({ text }: { text: string }) {
+    return (
+        <>
+            <button className="rounded-none bg-ablue m-auto p-3 font-medium text-lg text-awhite justify-center">{text}</button>
+        </>
```

Comment:
### children
When writing components, we usually want to create the best experience for our developers using the component. 

This ultimately comes down to the team's preference. I don't think there is a right or wrong way of doing it.

With that in mind, the React team created a default props value named `children`.
It essentially allows us to handle text values in a more intuitive way. 

For example: 
**without `props.children`**
```js
<Component text="text value" />
```

**with `props.children`**
```js
<Component>text value</Component>
```

The use of `props.children` is usually what you'll see being used in other codebases - due to how it resembles the already known HTML way of doing things and provides default support for nested elements.
 
**Here's an example of nested elements:**
Let's say we wanted to create a paragraph, but it also has an anchor tag inside it. 

```html
<p>A bunch of text with a <a>link</a></p>
```

If we were to use the custom `text` prop we wouldn't be able to recreate this structure. But `children` would allow this to be done.

```js
<Paragraph>A bunch of text with a <a>link</a></Paragraph>
```